docs/docs/api/LayoutAnimations/KeyframeAnimations.md

```diff
@@ -32,10 +32,30 @@ const keyframe = new Keyframe({
   }
 ```
 
+Instead of using '0' and '100', you can define edge points using `from` and `to` keywords. The result will be the same.
```

Comment:
To be consistent with the numbers above.

```suggestion
Instead of using `0` and `100`, you can define edge points using `from` and `to` keywords. The result will be the same.
```